solutions/33.java

```diff
@@ -0,0 +1,14 @@
+class Solution {
```

Comment:
Well, there is a scope of improvement, the very fact that the array is sorted, you can utilize binary search. Let me know if you are willing to add that solution , or i will merge this one .

Reply:
If someone wants to improve 33, they can. im not willing to do it.